app/controllers/answers-controller.rb

```diff
@@ -1,3 +1,19 @@
+post '/questions/:qid/answers' do
+  if !logged_in?
+    redirect '/users/login'
+  end
+  p 'YAYAYAYAYAYAYAYAYAYAYAY'
+  p params
+  @answer = Answer.new(params[:answer])
+  p @answer
+  if @answer.save
+    p 'saved!'
+    @question = Question.find(params[:qid])
```

Comment:
This isn't necessary (10-11) because the associations know about their inverses @laurieskelly